python/rmm/docs/guide.md

```diff
@@ -67,6 +67,38 @@ host:
 array([1., 2., 3.])
 ```
 
+## Prefetching DeviceBuffers Backed by Managed Memory
+
+CUDA Unified Memory, also known as managed memory, can be allocated using
+rmm.ManagedMemoryResource explicitly, or by calling `rmm.reinitialize` with
+`managed_memory=True`.
+
+A DeviceBuffer backed by managed memory may be prefetched to a specified
```

Comment:
```suggestion
A `DeviceBuffer` backed by managed memory may be prefetched to a specified
```

Reply:
Sorry, I was thinking doxygen where known names are automatically highlighted. I noticed there are other uses of DeviceBuffer in this file not quoted, will have a look.

Reply:
I found some more instances of unquoted MemoryResource and DeviceBuffer (notably when plural) and fixed them.